app/geo/gui/src/ts/twxplore/gui/geo/components/map/MapLayout.tsx

```diff
@@ -175,6 +176,21 @@ const MapLayoutImpl: React.FunctionComponent = () => {
 
             <SelectionPanel />
           </Grid>
+
+          <Grid
+            container
+            item
+            spacing={0}
+            direction="column"
+            className={classes.drawerGridContainer}
+            justify={"flex-start"}
+          >
+            <Grid>
+              <Typography align={"center"}>Toggle Layer Type</Typography>
+            </Grid>
+
+            <ToggleLayerPanel />
```

Comment:
Needs to be surrounded with a Grid item?

Reply:
ToggleLayerPanel returns a Grid item, so I don't think so.

Reply:
Usually better for components not to contribute to an "outer" context, in this case a Grid.

Reply:
Makes sense. I was doing that for my other panels so I'll see if I can fix that right now.